### PR TITLE
0.14.0

### PR DESCRIPTION
**Pull request by euri10**
WIP

- [x] Still need to decide what we do with websockets supported version(s), see #929, I'd be for dropping 8.* on that minor bump, but would prefer to hear others thoughts since the change is only related to a tooling PR !
- [ ] We could maybe squeeze in #1048 since it's related to #995 we add here, 
- [x] anything else ?

### CODE REVIEW

**Comment by Kludex** [comment]
I'm going to check https://github.com/encode/uvicorn/pull/1048 tonight.
I've replied https://github.com/encode/uvicorn/pull/929.

> anything else ?

I don't think so...

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+- Test logging (#964) 5/29/21, 7:07 AM 3e1e33eb
+- fix: add mypy on watchgodreload.py (#993) 5/29/21, 7:03 AM dfcad719
+- fix: add mypy on logging.py (#1003) 5/29/21, 6:50 AM d3655473
+- fix: add mypy on asgi2.py (#1000) 5/29/21, 6:38 AM e30bf135
+- fix: add mypy on basereload.py (#994) 5/29/21, 4:40 AM 3129c526
+- coverage: pin higher coverage (#1006) 5/28/21, 10:18 PM 4c485cc9
+- tests: ignore coverage on helper test functions (#980) 5/28/21, 7:26 PM abde790a
+- Add docs warning about reload and debug flags (#949) (#1045) (#1047) 5/28/21, 2:30 PM d81b96ee
+- Update docs about reload_dirs (#1028) 5/28/21, 11:23 AM 3542cb61
+- Change websockets version to v9 (#1038) 5/27/21, 3:00 PM 5314a900
```
Comment (Kludex) [inline CHANGELOG.md]: This one should not be skipped, afaik.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
+- ♻️ Replace _types module by asgiref.typing (#1044) 5/27/21, 3:00 PM 926a8f5d
+- 📝 Update markdown extensions to highlight code (#1043) 5/27/21, 10:04 AM d28f6835
+- Moved FlowControl class and globals to its own file in uvicorn/protocols/http/ (#1034) 5/27/21, 9:28 AM 21ff87f1
+- Fix PEP 508 violation (#1035) 5/24/21, 4:21 PM 0c9393ac
+- Allow Click 8.0, refs #1016 (#1042) 5/23/21, 6:00 AM 2c06f6dd
```
Comment (Kludex) [inline CHANGELOG.md]: Why do you think those two should not be included?

Comment (euri10) [reply CHANGELOG.md]: mmm not sure what you mentionned, is that the click and websockets you're referring to, if yes you're right !

Comment (Kludex) [reply CHANGELOG.md]: Yeah, those two.

**Review by Kludex — APPROVED** [review]

**Comment by euri10** [comment]
> I'm going to check #1048 tonight.
> I've replied #929.
> 
> > anything else ?
> 
> I don't think so...

updated the changelog, would be great to push in #929 at least, when you got time @Kludex

**Pull request merged**